subresource-loading.bs

```diff
@@ -31,7 +43,143 @@ specification describes how web browsers load those resources. It is expressed
+
+# Fetch web bundle # {#fetch-web-bundle-section}
+
+<div algorithm>
+To <dfn>fetch web bundle</dfn> given [=web bundle=] |web bundle| and [=fetch params=] |fetch params|:
+
+1. Assert:  |webbundle|'s [=web bundle/state=] is "fetching".
+
+1. Let |request| be |fetch params|'s [=request=].
+
+1. Let |request|'s [=request/url=] is |webbundle|'s [=web bundle/rule=]'s [=bundle rule/source=].
+
+     Note: Source URL is resolved on document's base URL.
+
+1. Let |request|'s [=request/destination=] is "webbundle",
+
+1. Let |request|'s [=request/mode=] is "cors",
+
+1. Let |request|'s [=request/credentials mode=] be |web bundle|'s [=web bundle/rule=]'s [=bundle rule/credentials=] if [=bundle rule/credentials=] is non-null. Otherwise "same-origin"
+
+1. Append a [=header=], a tuple of ("Accept", "application/webbundle;v=1"), to |request|'s [=request/header list=].
+
+    Note: Chromium's experimental implementation uses "application/webbundle;v=b2" as a header value.
+    Once the web bundle format ([[draft-yasskin-wpack-bundled-exchanges]]) goes RFC, Chromium would use "1" as a version number.
+
+1. Let |response| be the result of running <a spec="fetch">HTTP-network-or-cache fetch</a> given |fetch params|.
+
+    Note: Chromium's current implementation doesn't allow a *nested bundle*.
+    A Web bundle is never fetched from other web bundles.
+
+1. If |response|'s [=response/status=] is 200,
+
+    1. Let |webbundle|'s [=web bundle/fetched bundle=] be the result of parsing |response|'s body ([[draft-yasskin-wpack-bundled-exchanges]]) and |webbundle|' [=web bundle/state=] be "fetched". If parsing fails, or any other conformance is violated, let [=web bundle/fetched bundle=] be null and [=web bundle/state=] be "failed".
+
+        Note: In parsing, Chromium's experimental implementation only accepts "b2" as a web bundle format version number ([[draft-yasskin-wpack-bundled-exchanges]]).
+        Once the web bundle format goes RFC, Chromium would accept "1" as a web bundle format version number.
+
+1. Otherwise, let |webbundle|' [=web bundle/state=] be "failed".
+
+</div>
+
+# HTML monkeypatches  # {#html-monkeypatches}
+
+## Script ## {#script}
+
+Note: TODO. This section uses [=fetch web bundle=].
+
+# Fetch monkeypatches # {#fetch-monkeypatches}
+
+## Fetch subresource from web bundle  ## {#fetch-subresource-from-web-bundle}
+
+In <a spec="fetch">HTTP-network-or-cache fetch</a>, before
+
+> 8.22. Set httpCache to the result of determining the HTTP cache partition, given|httpRequest|.
+
+add the following steps:
+
+22.  Set the |response| to the result of [=fetch from web bundle=], given |httpRequest|.
+
+    1. If |response| is [=network error=], return [=network error=].
+
+    2. If |response| is non-null, skip the steps 8.22-8.24 and goto the step 9.
+
+        Note: That means a subresource from a webbundle never interacts with HttpCache.
+        We plan to support HttpCache as a feature enhancement in the future.
+
+<div algorithm>
```

Comment:
Can these algorithms be moved to "Algorithms" section?

Reply:
Done. Move algorithms into Algorithms section (except for monkeypatches).